osclib/stagingapi.py

```diff
@@ -1425,6 +1425,17 @@ def create_adi_project(self, name, use_frozenlinks=False, src_prj=None):
                  <arch>x86_64</arch>
               </repository>"""
 
+        reproducible_builds_repos = f"""
+               <repository name="rb_future1y">
+                 <path project="{name}" repository="standard"/>
+                 <arch>x86_64</arch>
+               </repository>
+               <repository name="rb_j1">
```

Comment:
so that's testing with `-j1` ? how much slowdown is that causing?

Reply:
https://rb.zq1.de/compare.factory/reproducible.json has `duration` values for -j1 and -j4 .
For most smaller packages, it does not make much of a difference, because non-build times make up for the majority of time. 
For MozillaFirefox, it says
```
      "build_duration"  : 39489,
      "build_duration4" : 22782,
```
and for bash
```
      "build_duration"  : 363,
      "build_duration4" : 274,
```
However, nothing needs to block on this build to finish.